src/services/gapi.js

```diff
@@ -0,0 +1,47 @@
+import $S from 'scriptjs';
+import once from 'lodash/once';
+import config from '../config';
+import {GOOGLE_SCOPES} from './appFirebase';
+
+const DISCOVERY_DOCS = ['https://www.googleapis.com/discovery/v1/apis/classroom/v1/rest'];
+const GOOGLE_SCOPES_STRING = GOOGLE_SCOPES.join(' ');
```

Comment:
I don’t feel strongly but I’m not sure the string version needs to be its _own_ constant (vs. just putting `GOOGLE_SCOPES.join(' ')` on line 35)